plugins/services/src/js/components/RecentOffersSummary.js

```diff
@@ -10,170 +10,166 @@ const displayedResourceValues = {
   roles: 'Role',
   constraints: 'Constraints',
   cpus: 'CPU',
-  mem: 'Mem',
+  mem: 'Memory',
   disk: 'Disk',
   ports: 'Ports'
 };
 
-const getColumnClassNameFn = (classes) => {
-  return (prop, sortBy) => {
-    return classNames(classes, {
-      'active': prop === sortBy.prop
-    });
-  };
+const maxBarHeight = 200;
```

Comment:
I know we haven't discussed that but I try to go `MAX_BAR_HEIGHT` for constant top level constants.